### DISCUSSION TITLE
ComicTagger not updating ComicInfo.xml?

### DISCUSSION THREAD

**Discussion by michalorestes** [comictagger/comictagger#682] [Q&A]
As in the title, ComicTagger doesn't update any information stored in the ComicInfo.xml file after clicking on "Save Tags". Is this a expected behavior? Most comic readers read metadata from ComicInfo.xml so this is a little surprising.

**Comment by lordwelch** [comment]
Use https://github.com/comictagger/comictagger/releases/tag/1.6.0-beta.1
